vnets.tf

```diff
@@ -195,6 +195,13 @@ resource "azurerm_subnet" "public_vnet_ci_jenkins_io_agents" {
   virtual_network_name = azurerm_virtual_network.public.name
   address_prefixes     = ["10.245.2.0/23"] # 10.245.2.1 - 10.245.3.254
 }
+resource "azurerm_subnet" "public_jenkins_sponsorship_vnet_ci_jenkins_io_agents" {
+  provider             = azurerm.jenkins-sponsorship
+  name                 = "${azurerm_virtual_network.public_jenkins_sponsorship.name}-ci_jenkins_io_agents"
+  resource_group_name  = azurerm_resource_group.public_jenkins_sponsorship.name
+  virtual_network_name = azurerm_virtual_network.public_jenkins_sponsorship.name
+  address_prefixes     = ["10.200.2.0/24"] # 10.200.2.1 - 10.200.3.254
```

Comment:
Is it normal that this number of IPs is not the same as the one you give in the pull request description?

Reply:
Nevermind, misreaded